server.js

```diff
@@ -0,0 +1,55 @@
+
+const express = require('express');
+const bodyParser = require('body-parser');
+const mysql = require('mysql');
+const app = express();
+const cors = require('cors');
+
+app.use(cors());
+
+
+app.use(bodyParser.json());
+
+
+var con = mysql.createConnection({
+    host:'localhost',
+    user:'root',
+    password:'Infobeans',
+    database : 'todolist'
```

Comment:
Let's store these credentials in a separate dbCredentials.json file and use it.

Reply:
Separate DB config is created